packages/atomic/src/components/result-template-components/atomic-result-link/atomic-result-link.tsx

```diff
@@ -56,16 +59,48 @@ export class AtomicResultLink implements InitializableComponent {
   @Prop({reflect: true}) hrefTemplate?: string;
 
   private interactiveResult!: InteractiveResult;
-  private hasSlot!: boolean;
+  private hasDefaultSlot!: boolean;
+  private linkAttributes?: Attr[];
 
   public initialize() {
     this.interactiveResult = buildInteractiveResult(this.bindings.engine, {
       options: {result: this.result},
     });
   }
 
+  private isAttributeSlot(element: Element): element is HTMLAnchorElement {
+    return element.slot === 'attributes' && element.nodeName === 'A';
```

Comment:
Should there be some warning if someone is trying to use a `slot="attributes"` with something other than an anchor ?